### ISSUE TITLE
Various accessibility issues

### ISSUE THREAD

**Issue by davidbmaier** [openplanet-nl/tmio#17]
Just a list of obvious a11y issues I found while going through the main tm.io pages.

- [ ] TMGL and Nadeo player color is too bright in light mode. Same also applies to the TMGL tag and the text inside (the Nadeo tag text is black so that's fine).
![](https://user-images.githubusercontent.com/17618532/168292382-ff56b188-d1e8-41b5-8dc1-ff3faed6b9f0.png)
![](https://user-images.githubusercontent.com/17618532/168294321-3059a05e-05ba-4a56-9ac4-0c7a07a806e5.png)
![](https://user-images.githubusercontent.com/17618532/168292533-52049b2e-822b-47d5-afcf-b0bc1c6fcf05.png)
- [ ] Club tags can contain any color, so they can run into bad contrast as well - dark colors are especially bad in light mode.
![](https://user-images.githubusercontent.com/17618532/168292663-1f5f1eb2-76cc-4847-bbf8-afeb62a0afdc.png)
- [ ] Club, club room, campaign and map names can have very low contrast in light mode - in dark mode, dark colors are automatically inverted, but the same doesn't happen in light mode for especially bright colors.
![](https://user-images.githubusercontent.com/17618532/168294941-04ceaee7-1d3f-4035-8269-2d50bd7acbb4.png)
- [ ] Some additional text elements don't have enough contrast in light mode - the "Unplayed" one is a bit of an edge case (since it's just a non-interactive placeholder), but it's still very bright.
![](https://user-images.githubusercontent.com/17618532/168293538-a8140d02-3b76-410e-8820-cf4e04a9db2b.png)
![](https://user-images.githubusercontent.com/17618532/168293682-1f5cf010-a026-40a0-8ee1-1743b4f97c20.png)
![](https://user-images.githubusercontent.com/17618532/168294141-29e6dbec-b07f-4176-b818-41a21849b060.png)
![](https://user-images.githubusercontent.com/17618532/168295207-e3831853-be64-46e0-b814-91df0b37dc3d.png)
![](https://user-images.githubusercontent.com/17618532/168295647-2e8964ea-b39a-4f0d-aacf-17ec71f6b4d7.png)
![](https://user-images.githubusercontent.com/17618532/168295710-10fa512f-3afa-4d3a-8920-6214111e71c0.png)
![](https://user-images.githubusercontent.com/17618532/168293857-8fe6991c-ebe2-41a0-b8b7-96353d89002c.png)

There are some additional very minor ones that I didn't list here because they're pretty close to WCAG guidelines, and some are just based on brand guidelines (like white text on Twitter-blue).

**Comment by davidbmaier** [comment]
Closing due to the big UI rework - most of these are invalid now.
